specification/translation/Azure.AI.DocumentTranslation/routes.tsp

```diff
@@ -0,0 +1,437 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace BatchDocumentTranslationClient;
+
+interface DocumentTranslationOperations {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+  @summary("Submit a document translation request to the Document Translation service")
+  @doc("""
+Use this API to submit a bulk (batch) translation request to the Document
+Translation service.
+Each request can contain multiple documents and must
+contain a source and destination container for each document.
+            
+The
+prefix and suffix filter (if supplied) are used to filter folders. The prefix
+is applied to the subpath after the container name.
+            
+Glossaries /
+Translation memory can be included in the request and are applied by the
+service when the document is translated.
+            
+If the glossary is
+invalid or unreachable during translation, an error is indicated in the
+document status.
+If a file with the same name already exists at the
+destination, it will be overwritten. The targetUrl for each target language
+must be unique.
+""")
+  @route("/batches")
+  @post
+  startTranslation is Azure.Core.Foundations.Operation<
+    StartTranslationDetails,
+    void
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Doesn't fit standard ops"
+  @summary("Returns a list of batch requests submitted and the status for each request")
+  @doc("""
```

Comment:
For these really lengthy doc comment and summary, I suggest move to a separate file and use augment decorator style (@@doc) to keep the code easier to read.